mdpow/workflows/dihedrals.py

```diff
@@ -426,30 +487,38 @@ def periodic_angle(df, padding=45):
 
     return df_aug
 
-def dihedral_violins(df, width=0.9, solvents=SOLVENTS_DEFAULT):
-    '''Plots distributions of dihedral angles for one dihedral atom group
-       as violin plots, using as input the augmented :class:`pandas.DataFrame`
-       from :func:`~mdpow.workflows.dihedrals.periodic_angle`.
+def dihedral_violins(df, width=0.9, solvents=SOLVENTS_DEFAULT, plot_title=None):
+    '''Plots kernel density estimates (KDE) of dihedral angle frequencies for
+       one dihedral atom group as violin plots, using as input the augmented
+       :class:`pandas.DataFrame` from :func:`~mdpow.workflows.dihedrals.periodic_angle_padding`.
+
+       Output is converted to SVG by :func:`~mdpow.workflows.dihedrals.build_svg`
+       and final output is saved as PDF by :func:`~mdpow.workflows.dihedrals.plot_dihedral_violins`
 
        :keywords:
 
        *df*
            augmented results :class:`pandas.DataFrame` from
-           :func:`~mdpow.workflows.dihedrals.periodic_angle`
+           :func:`~mdpow.workflows.dihedrals.periodic_angle_padding`
 
        *width*
            width of the violin element (>1 overlaps)
            default: 0.9
 
        *solvents*
            The default solvents are documented under :data:`SOLVENTS_DEFAULT`.
```

Comment:
document that this is normally a 2-tuple; state what happens when it only contains a single solvent